swatch-billable-usage/src/main/java/com/redhat/swatch/billable/usage/services/BillableUsageService.java

```diff
@@ -135,7 +144,7 @@ public BillableUsage produceMonthlyBillable(BillableUsage usage)
     return usage;
   }
 
-  private ContractCoverage getContractCoverage(BillableUsage usage)
+  protected ContractCoverage getContractCoverage(BillableUsage usage)
```

Comment:
Why the scope of this method changed to protected?

Reply:
Something I changed for a test, then didn't need.